### DISCUSSION TITLE
Cancelled workflows are still in in_progress actions list

### DISCUSSION THREAD

**Discussion by sandynz** [community/community#31690] [Actions] [answered]
Hi, I found cancelled workflows are still in in_progress actions list, screenshot from https://github.com/apache/shardingsphere/actions?query=is%3Ain_progress:
<img width="975" alt="图片" src="https://user-images.githubusercontent.com/42492540/188054322-f3338cf6-422b-41ae-8a60-746f2946fce7.png">

These workflows started several days ago, and the detail page shows it's already failed, from https://github.com/apache/shardingsphere/actions/runs/2925896997:
`Canceling since a higher priority waiting request for 'check-license-refs/pull/20508/merge' exists`
<img width="771" alt="图片" src="https://user-images.githubusercontent.com/42492540/188055430-ded7fbe7-8d7d-4c90-8161-176b63237530.png">

I tested one workflow, re-ran failed jobs and then cancel workflow, it'll disappear from in_prgress actions list.

I have some questions about it:
1. Are these cancelled workflows still consume maximum concurrent jobs (it's limited)?
2. Is there better way to clean these cancelled workflows?
3. Is it a bug?

Thanks

**Comment by lhotari** [comment, marked as answer]
We have a similar issue in Apache Pulsar, https://github.com/apache/pulsar/actions?query=is%3Ain_progress . 
In addition to having cancelled workflows in the in_progress list, we don't see all jobs that are executing. 

I created an issue to support.github.com requesting a fix for this. We have had cases in the past where the GitHub Actions search index got corrupted for apache/pulsar and GitHub admin fixed it somehow with a manual admin operation behind the scenes.
I haven't yet received a reply to my issue, but I would assume that this is something similar.

**Reply by taojintianxia** [reply]
hi there, would you please send me the link of the issue you created to github admin ? we got the same problem, there are a lot of action blocked in queue.

**Reply by lhotari** [reply]
Support tickets aren't public so I don't think that it would be helpful to share the link.

**Reply by lhotari** [reply]
@taojintianxia what is your repository where you see the problem?

**Reply by taojintianxia** [reply]
> @taojintianxia what is your repository where you see the problem?

hi there, I'm the contributor from Apache ShardingSphere. and I rised an issue to github action runner. Today I got the answer : there is an incident for github action cause the queued task:

https://www.githubstatus.com/incidents/pz0bvrjx9ztp

**Reply by sandynz** [reply]
Thanks
